src/csync/csync_exclude.cpp

```diff
@@ -330,8 +323,17 @@ bool ExcludedFiles::reloadExcludeFiles()
     bool success = true;
     const auto keys = _excludeFiles.keys();
     for (const auto& basePath : keys) {
-        for (const auto& file : _excludeFiles.value(basePath)) {
-            success = loadExcludeFile(basePath, file);
+        for (const auto& excludeFile : _excludeFiles.value(basePath)) {
+            QFile file(excludeFile);
+            if (file.open(QIODevice::ReadOnly)) {
+                if (file.exists()) {
+                    loadExcludeFilePatterns(basePath, file);
+                } else {
+                    success = false;
```

Comment:
Do we ever get in there? If the file doesn't exist I'd expect the `open(QIODevice::ReadOnly)` to have failed and returned false already.